packages/nebula/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@thirdweb-dev/nebula",
+  "version": "1.0.0",
```

Comment:
maybe start at `0.0.0` then add a first changeset that bumps it to `minor` meaning the first release will be `0.1.0`